Homeworks/src/Week1/Week1Task2.java

```diff
@@ -1,4 +1,35 @@
 package Week1;
 
 public class Week1Task2 {
+  public static void main(String[] args) {
+
+        System.out.print("Enter first number: ");
+        Scanner first =  new Scanner(System.in);
+       int firstNumber = first.nextInt();
+
+        System.out.print("Enter second number: ");
+        Scanner second =new Scanner(System.in);
```

Comment:
```suggestion
        <delete other instances, keep only 1>
```
`Scanner `objesini bir kere çağırdıktan sonra tekrardan çağırmamıza gerek yok.